packages/x-utils/tsconfig.esm.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Any reason to `target` another `ECMAScript` version when building it as a `ESmodule` instead of using the same `target` as in the `commonjs` one?